spec/controllers/dashboards_controller_spec.rb

```diff
@@ -148,36 +148,31 @@ def load_table(klass, options)
       end
 
       context 'when successful' do
-        it 'adds a new version record' do
+        it 'sets the new production version' do
           SiteMapperHelper.silence do
-            expect { post(:push) }.to change(Version, :count).by(1)
+            expect { post(:push) }.to change(Version, :count).by(0)
```

Comment:
Given that it `sets the new production version`, can we have an expectation to test that `version.production` changes from `false` to `true`?

Reply:
Updated the test to expect that `current_production` is nil before `push()`, and that `current_production` is equal to `current_preview` and that `production` is `true`.